static/rules-engine/index.css

```diff
@@ -0,0 +1,182 @@
+html, body {
+  margin: 0;
+  padding: 0;
```

Comment:
For the rest of the gateway we set a base 10px font size and then use rem rather than px for sizing everything. The reason for this is to make it easier to adjust to screens of different pixel densities by just changing the base font size. Currently all the rules engine dimensions are set in px and it doesn't scale well on mobile.